src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/ProviderBase/DbConnectionInternal.cs

```diff
@@ -203,6 +203,14 @@ virtual protected bool ReadyToPrepareTransaction
             }
         }
 
+        virtual internal bool IsAccessTokenExpired
```

Comment:
can we fix it to` internal virtual` please?

Reply:
I also see there's a lot of such inconsistent patterns.. will need a separate PR for all this. But I'll fix this one.